tests/Support/SupportCollectionTest.php

```diff
@@ -4298,7 +4298,7 @@ public function testWhenEmpty($collection)
     {
         $data = new $collection(['michael', 'tom']);
 
-        $data = $data->whenEmpty(function ($collection) {
+        $data = $data->whenEmpty(function ($data) {
             return $data->concat(['adam']);
         });
```

Comment:
This was never caught, because it's never executed :rofl: 

Might be better to just throw from in there:

```suggestion
        $data = $data->whenEmpty(function () {
            throw new Exception("Expected whenEmpty callback not to be called");
        });

Reply:
Sure, I like this approach.